src/components/pager/__internal__/pager-navigation.component.tsx

```diff
@@ -161,24 +168,33 @@ const PagerNavigation = ({
   return (
     <StyledPagerNavigation>
       {!hasOnePage && renderButtonsBeforeCount()}
-      {showPageCount && (
-        <StyledPagerNavInner>
-          <label htmlFor={currentPageId}>
-            <StyledPagerNoSelect>{l.pager.pageX()}</StyledPagerNoSelect>
-          </label>
-          <NumberInput
-            value={currentPage.toString()}
-            data-element="current-page"
-            onChange={handleCurrentPageChange}
-            onBlur={handlePageInputChange}
+      {showPageCount &&
+        (!nonInteractivePageNumber ? (
+          <StyledPagerNavInner>
+            <label htmlFor={currentPageId}>
```

Comment:
This was what I was referring to when I said could we make this a label instead of a div within label etc, will it not work as just a `label` instead of `StyledPagerNoSelect` rendering a `div`?